test/Tests.Infrastructure/RavenTestBase.Sharding.cs

```diff
@@ -51,19 +51,19 @@ public DocumentStore GetDocumentStore(Options options = null, [CallerMemberName]
             {
                 r.Sharding ??= new ShardingConfiguration();
 
-                if (shards == null)
+                if (shards == null && r.Sharding.Shards == null)
```

Comment:
what wasn't working here?

Reply:
now we can pass Shards topology in ShardingConfiguration